platforms/common/include/px4_platform_common/param.h

```diff
@@ -73,7 +73,7 @@ inline static param_t param_handle(px4::params p)
 
 // Define a list of parameters. This macro also creates code to update parameters.
 // If you get a compile error like:
-//   error: virtual function ‘virtual void <class>::updateParamsImpl()’
+//   error: virtual function 'virtual void <class>::updateParamsImpl()'
```

Comment:
why did this change? we need to avoid making superficial changes to the fork

Reply:
there are a few examples of this actually

Reply:
This is actually a fix for cppcheck errors. The report fails to generate due to encoding. These characters are outside of the "normal" (using that very loosely) ASCII encoding the report is using.